site/plugins.md

```diff
@@ -467,6 +467,8 @@ In particular, a node may contain:
 - `latest` - an id-value map containing the latest values. Each id has only one value.
 - `latestControls` - the latest available controls.
 - `metrics` - the collection of metrics to display in the UI. Each metric has multiple timestamped values.
+- `sets` - a string->set-of-strings map, for example a list of local networks.
+- `counters` - a string->int map.
```

Comment:
Sorry to keep adding things, but each time I look I find something else worth mentioning.

We should mention at least somewhere about the different behaviours when merging the same node from reports over multiple machines:
* `latest` maps get merged, with conflicting values being resolved by the latest timestamp
* `sets` get joined as a union
* `counters` get merged additively, ie. if the same key is present in both then the final result will be the sum of the individual values.